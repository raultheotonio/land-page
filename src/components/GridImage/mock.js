export default {
  background: false,
  title: 'My Grid Image',
  description: 'Description',
  grid: [
    {
      altText: 'alt text 1',
      srcImg: 'https://source.unsplash.com/random/800x800?r=1',
    },
    {
      altText: 'alt text 2',
      srcImg: 'https://source.unsplash.com/random/800x800?r=2',
    },
    {
      altText: 'alt text 3',
      srcImg: 'https://source.unsplash.com/random/800x800?r=3',
    },
    {
      altText: 'alt text 4',
      srcImg: 'https://source.unsplash.com/random/800x800?r=4',
    },
    {
      altText: 'alt text 5',
      srcImg: 'https://source.unsplash.com/random/800x800?r=5',
    },
  ],
};
