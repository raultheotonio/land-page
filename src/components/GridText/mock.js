export default {
  title: 'title',
  description: 'description',
  background: false,
  grid: [
    {
      title: 'grid 1',
      description: 'grid description',
    },
    {
      title: 'grid 2',
      description: 'grid description',
    },
    {
      title: 'grid 3',
      description: 'grid description',
    },
  ],
};
