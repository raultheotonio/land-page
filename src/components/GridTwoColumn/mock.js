export default {
  title: 'Grid two columns',
  text: 'Somente um teste aqui por favor',
  background: true,
  srcImg: 'assets/images/javascript.svg',
};
