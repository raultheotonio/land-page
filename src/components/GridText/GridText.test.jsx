import { GridText } from '.';
import renderTheme from '../../utils/RenderTheme';
import mock from './mock';

describe('<GridText />', () => {
  it('should render', () => {
    const { container } = renderTheme(<GridText {...mock} />);

    expect(container).toMatchSnapshot();
  });
});
