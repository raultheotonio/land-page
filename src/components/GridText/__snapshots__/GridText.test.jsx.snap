// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GridText /> should render 1`] = `
<div>
  <div
    class="sc-jSUZER kdiEaK"
  >
    <div
      class="sc-gKPRtg fxGKZy"
    >
      <div
        class="sc-dkrFOg hJNvJK"
      >
        <div>
          <h2
            class="sc-bcXHqe icqeWr"
          >
            title
          </h2>
        </div>
        <p
          class="sc-gswNZR ddvriM"
        >
          description
        </p>
        <div
          class="sc-hLBbgP lfBapZ"
        >
          <div
            class="sc-eDvSVe hrYwgi"
          >
            <div>
              <h3
                class="sc-bcXHqe khFHLM"
              >
                grid 1
              </h3>
            </div>
            <p
              class="sc-gswNZR ddvriM"
            >
              grid description
            </p>
          </div>
          <div
            class="sc-eDvSVe hrYwgi"
          >
            <div>
              <h3
                class="sc-bcXHqe khFHLM"
              >
                grid 2
              </h3>
            </div>
            <p
              class="sc-gswNZR ddvriM"
            >
              grid description
            </p>
          </div>
          <div
            class="sc-eDvSVe hrYwgi"
          >
            <div>
              <h3
                class="sc-bcXHqe khFHLM"
              >
                grid 3
              </h3>
            </div>
            <p
              class="sc-gswNZR ddvriM"
            >
              grid description
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
